Modules/IxNetRestApiFileMgmt.py

```diff
@@ -376,25 +260,10 @@ def importJsonConfigObj(self, dataObj, option='modify', silentMode=False, timeou
         if option is 'newConfig':
             arg3 = True
 
-        dataReformatted = {"arg1": "{0}/ixnetwork/resourceManager".format(self.ixnObj.headlessSessionId),
-                           "arg2": json.dumps(dataObj),
-                           "arg3": arg3}
-
-        url = self.ixnObj.sessionUrl+'/resourceManager/operations/importconfig'
-        response = self.ixnObj.post(url, data=dataReformatted, silentMode=silentMode)
-        response = self.ixnObj.waitForComplete(response, url+'/'+response.json()['id'], silentMode=False, timeout=timeout)
-        if response.json()['result'] != []:
```

Comment:
Timeout is handled in RestAPI, but missed to handle RestPy implementation

Reply:
Timeout handled internally in Restpy

Reply:
Please confirm, if RestPy can take the configured timeout, as this timeout will be configured from user